README.md

```diff
@@ -15,11 +15,11 @@ Boost.CI also allows you to run a big-endian build on Travis CI.
 Here are all the steps you need to take as a Boost repository maintainer to enable all of these CI features in your repository:
 
 1. Checkout `develop` and then make a new branch called `ci`.
-1. Copy the `template/appveyor.yml` file from this repository into the *top level* of your repository. You can also rename name it as `.appveyor.yml`.
-1. Copy the `template/azure-pipelines.yml` file from this repository into the top level or your repository. You can also rename name it as `.azure-pipelines.yml`.
-1. Copy the `template/.travis.yml` file from this repository into the top level or your repository.
+1. Copy the `.appveyor.yml` file from this repository into the *top level* of your repository.
+1. Copy the `.azure-pipelines.yml` file from this repository into the top level or your repository.
+1. Copy the `.travis.yml` file from this repository into the top level or your repository.
 1. Copy the `LICENSE` file from this repository to the top level of your repository.  This adds the `BSL-1.0` designation to your repository on github.
-1. [optional] Copy the `template/README.md` file from this repository to the top level of your repository.  If you already have a README.md then you can take what you need from the template version to improve it, if desired.  Otherwise, you will need to customize README.md for your repository.  One useful step is to fixup the repository name using the command `sed -i 's/template/<myrepositoryname>/g' README.md`, and then update the first line description.
+1. [optional] Copy the `templates/README.md` file from this repository to the top level of your repository.  If you already have a README.md then you can take what you need from the template version to improve it, if desired.  Otherwise, you will need to customize README.md for your repository.  One useful step is to fixup the repository name using the command `sed -i 's/template/<myrepositoryname>/g' README.md`, and then update the first line description.
```

Comment:
I'd move `/template/README.md` to `/README.template.md`

Reply:
Done :)

Reply:
Thx